drivers/video/video_esp32_dvp.c

```diff
@@ -0,0 +1,463 @@
+/*
+ * Copyright (c) 2024 espros photonics Co.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT espressif_esp32_cam
+
+#include <soc/gdma_channel.h>
+#include <soc/lcd_cam_reg.h>
+#include <soc/lcd_cam_struct.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/dma.h>
+#include <zephyr/drivers/dma/dma_esp32.h>
+#include <zephyr/drivers/interrupt_controller/intc_esp32.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/video.h>
+#include <zephyr/irq.h>
+#include <zephyr/kernel.h>
+#include <zephyr/pm/device.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(esp32_cam, LOG_LEVEL_INF);
+
+#define VIDEO_ESP32_F160M               160000000UL
+#define VIDEO_ESP32_DMA_BUFFER_MAX_SIZE 4095
```

Comment:
These symbols can be properties in the dts bindings and selected per-project via dts/overlay as well.

Reply:
Same as above.